packages/cli-lib/examples/slack/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think this is handled automatically by the top-level `.gitignore`

Reply:
I had it in there in case someone wants to copy the code as a boilerplate, I can take it out thought